In [9]:
import time # Python 标准库中的 time 模块

In [10]:
tik1=time.time()
#返回当前时间的时间戳
# （从 1970 年 1 月 1 日 00:00:00 UTC 到现在的秒数，浮点数）
time.sleep(2)   #让程序暂停指定的秒数：time.sleep(secs)
tik2=time.time()
interval=tik2-tik1  #这里用来得到tik1与tik2之间的时间间隔
print(f'interval:{interval}')

interval:2.000857353210449


interval:2.0002033710479736

In [11]:
time.localtime(tik1) #将时间戳tik1转换为本地时间的结构化元组

time.struct_time(tm_year=2025, tm_mon=9, tm_mday=4, tm_hour=18, tm_min=11, tm_sec=59, tm_wday=3, tm_yday=247, tm_isdst=0)

上例返回的结果：time.struct_time(tm_year=2025, tm_mon=9, tm_mday=4, tm_hour=17, tm_min=44, tm_sec=46, tm_wday=3, tm_yday=247, tm_isdst=0)

In [12]:
time.localtime(tik1).tm_year  #2025

2025

In [13]:
format=time.strftime("%Y-%m-%d %H:%M:%S")
t=time.localtime(tik1)
time.strftime(format,t)
#   '2025-09-04 18:05:49'

'2025-09-04 18:12:01'

```python
将结构化时间（如 localtime() 返回的元组）格式化为字符串。常用格式符：
%Y：四位数年份
%m：月份
%d：日期
%H：小时
%M：分钟
%S：秒

